pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -49,6 +49,7 @@ let
       gssSupport = browser.gssSupport or false;
       alsaSupport = browser.alsaSupport or false;
       pipewireSupport = browser.pipewireSupport or false;
+      useOpenscPkcs11 = cfg.useOpenscPkcs11 or false;
```

Comment:
Is there any other provider of PKCS#11 other than OpenSC? If the answer is "no", then I'd like for this option to be renamed to `smartcardSupport`

Reply:
There are. For example, YubiKey. OpenSC, though, is the most often used.

Reply:
But does Firefox support something other than OpenSC?

Reply:
In theory yes, but, as far as I know, no one uses other methods on practice. For example, "PKCS #11" plugin provided by Estonian government already assumes that OpenSC is installed on the system and is in PATH.

Reply:
Cool, go ahead and rename the option to `smartcardSupport` then :)